core/unit_test/CMakeLists.txt

```diff
@@ -314,25 +314,13 @@ KOKKOS_ADD_TEST_EXECUTABLE(
 )
 # We need -rdynamic on GNU platforms for the stacktrace functionality
 # to work correctly with shared libraries
-if(NOT KOKKOS_HAS_TRILINOS)
-SET_PROPERTY(TARGET StackTraceTestExec PROPERTY ENABLE_EXPORTS 1)
+SET_PROPERTY(TARGET ${PACKAGE_NAME}_StackTraceTestExec PROPERTY ENABLE_EXPORTS ON)
 
-KOKKOS_ADD_TEST( NAME UnitTest_StackTraceTest_normal
-                 EXE  StackTraceTestExec
+KOKKOS_ADD_TEST( NAME UnitTest_StackTraceTest
```

Comment:
@dalg24 if I replace lines 319-22

```
KOKKOS_ADD_TEST( NAME UnitTest_StackTraceTest
                 EXE  ${PACKAGE_NAME}_StackTraceTestExec
                 FAIL_REGULAR_EXPRESSION "FAILED"
              )
```

with this:
```
KOKKOS_ADD_ADVANCED_TEST( UnitTest_StackTraceTest
                 TEST_0
                  EXEC  StackTraceTestExec
                  NUM_MPI_PROCS 1
                  FAIL_REGULAR_EXPRESSION "FAILED"
               )
```

then the tests build and pass in Trilinos:

```
...
      Start  5: KokkosCore_UnitTest_StackTraceTest
 1/24 Test  #5: KokkosCore_UnitTest_StackTraceTest ...............   Passed    0.06 sec
...
```

Reply:
This solution doesn't work with kokkos standalone, in that this removes the stacktrace test for standalone

https://github.com/kokkos/kokkos/blob/586d07ba08e2bb9aae6fe644cf7ac16853dd32bd/cmake/fake_tribits.cmake#L145-L151

`KOKKOS_ADD_ADVANCED_TEST` has no implementation outside of Trilinos usage (true of master used by Trilinos and VOTD develop)

Reply:
> if I replace lines 319-22

Why did you change these lines in the first place?  What was wrong with them?

Reply:
@jjwilke Would you please look into this

Reply:
Look into filling in the TODO? Basically you want an implementation of ADVANCED_TEST for Kokkos standalone?